content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -211,7 +209,9 @@ export DD_DOTNET_TRACER_HOME=/opt/datadog
 dotnet example.dll
 ```
 
-To set the environment variables on a Linux Docker container, use [`ENV`][2]:
+#### Linux Docker container
+
+To set the required environment variables on a Linux Docker container:
```

Comment:
It feels like it would be nice to have an encapsulated Docker image example that shows how to do everything. i.e. 

* Install Tracer
* Run `/opt/datadog/createLogPath.sh`
* Add environment variables

Currently you have to try and piece it all together for your environment, instead of just having e.g. 3 example Docker files (for debian/alpine/ubuntu - [the Docker Image base-OSs supported by Microsoft](https://hub.docker.com/_/microsoft-dotnet-aspnet/))

Maybe it makes more sense to have those as blog/post sample? But that's what I would be looking for when trying to get up and running quickly

Reply:
That is a good call out - maybe a good happy path would be to add an example Dockerfile to our Github / samples repo - then cross link this back? I am not sure about this one. @lucaspimentel thoughts?

Reply:
We have something... somewhere. Can't remember right now, but it's definitely not linked in the docs, which is a good idea.

Reply:
I'm going to go ahead and publish this PR without this, and when you're ready to add an example, we can open a clean PR.